data/systemd/snapd.service.in

```diff
@@ -3,8 +3,8 @@ Description=Snappy daemon
 Requires=snapd.socket
 
 [Service]
-ExecStart=/usr/lib/snapd/snapd
-EnvironmentFile=/etc/environment
+ExecStart=@libexecdir@/snapd/snapd
+EnvironmentFile=@SNAPD_ENVIRONMENT_FILE@
```

Comment:
This should probably be made non-mandatory, like we do for the Fedora units.

That is:
```
EnvironmentFile=-@SNAPD_ENVIRONMENT_FILE@
```